doc/Quick-Start-Guide.rst

```diff
@@ -146,6 +146,36 @@ Troubleshooting
        sudo systemctl restart lnet
        sudo lctl list_nids
 
+- If you see error::
```

Comment:
>The solution comes from https://github.com/Seagate/cortx/blob/main/doc/Build-Installation-FAQ.md

We don't duplicate the information everywhere.
Adding a reference to the doc is a better idea.

This is a quick start guide, and let's keep it as simple as possible.
We may have another document to list FAQ, troubleshooting, or other information.

Reply:
Thanks, that makes sense. 
I duplicated the solution because I saw Qs2 from that doc is already duplicated in the Quick Start Guide, so I wanted to follow the same style.

I agree that adding a reference to the doc will be a better idea. I will close this one and create another PR.
In that case, do you think we should remove the solution to "lctl list_nids command does not show any output" problem? It's also duplicated as Qs2 from the doc.

Reply:
I would prefer them to be kept untouched.
Because, developers may see this network issue before doing any testing. So they may be blocked by that.

But the problem you mentioned, only happens after a non-clean stop of tests. (if some tests fail and the kernel modules are not removed by the testing script)

Reply:
Got it thanks